SingularityUI/app/templates/vex/requestRun.hbs

```diff
@@ -24,9 +24,13 @@
     <input name="message" id="run-message" type="text" />
 </div>
 <hr class='border-color-light-gray'>
-
-<p><label class='label-light'><input id="autoTail" name="autoTail" type="checkbox"> Wait for task to start, then start tailing:</label></p>
+<strong>After triggering the run:</strong>
+<p>
+    <label class='label-light'><input id="stay-on-request-page" name="afterStart" type="radio" value="stay-on-request-page"> Stay on this request's page</label>
```

Comment:
I'm going to take out the word "request's" here because the dialog can be accessed from pages other than just a request page.